samples/AspNetCore/ControllerSample/Readme.md

```diff
@@ -136,13 +136,19 @@ On Windows:
 ```C#
  public void ConfigureServices(IServiceCollection services)
 {
-    services.AddControllers().AddDapr();
+    services.AddControllers().AddDapr(builder => 
+        builder.UseJsonSerializationOptions(
```

Comment:
We can remove the customized JSOn settings from the startup now that defaults are good enough.

Reply:
Moved JsonSerializer defaults to DaprClientBuilder (in d1c5447).